nih-sparc/sparc-app
# Anatomical map tags #4m8hgy

# Description

Refactor the behavior of clicking on the anatomical map to add a filter instead of updating the search query.  

## Type of change

Delete those that don't apply.


# How Has This Been Tested?

1. Open the anatomical map on the Find Data - Datasets tab.
2. Click on Heart
3. The results should be filtered, the Heart filter should be selected.
4. Open the map again, click on Heart again.
5. Nothing should change (make sure the query params in the address bar stay the same)
6. Open the map again, click on "Carotid body" (it's in the right part of the neck, you'll probably need to zoom in.)
7. Carotid body should be added.
8. Rinse and repeat - try to add Carotid body a second time and make sure nothing changes.

Please note that this change is dependent upon updating the new Contentful filter "Other Anatomic Elements" once we get a list from Alan.  If you click on an element that isn't yet a filter value, you're going to get weird results, so we need to make sure that list stays up to date.

# Checklist:
